tests/test_openclhpp.cpp

```diff
@@ -3626,7 +3630,63 @@ void testDevice_GetInfo_CLDeviceName()
     clGetDeviceInfo_StubWithCallback(clGetInfo_testDeviceGetInfoCLDeviceName);
     cl::string deviceName = devicePool[0].getInfo<CL_DEVICE_NAME>();
     TEST_ASSERT_EQUAL_STRING(expected.c_str(), deviceName.c_str());
-} 
+}
+
+#if defined(cl_ext_device_fission)
+static cl_int clCreateSubDevices_testDevice_createSubDevices(
+    cl_device_id in_device, const cl_device_partition_property *properties,
+    cl_uint num_devices, cl_device_id *out_devices, cl_uint *num_devices_ret,
+    int num) {
+    return 0;
+}
```

Comment:
This entire function looks like dead code.  If this is not used can it be removed?

Reply:
Sure, removed.